classes/scripts/runner.php

```diff
@@ -608,7 +606,7 @@ public static function enableAutorun($name)
 						{
 							$autorunner->writeError($exception->getMessage());
 
-							exit($exception->getCode());
+							exit(2);
```

Comment:
From the end-user point of view, the error type is irrelevant because the only important information is "it's not success (i.e 0) because test suite can not be executed".
So, i think that using exception code if it is not equal to 0 in this context is useless (however, put it in a technical log may be interesting for support, but atoum has no logger currently).
Moreover, using exception code imply collect and document all of them to avoid collision. Is it relevant to do this work instead of more usefull thing? I'm not really sure.
And maybe using 2 for error and 3 for exception is a bad idea, maybe using 2 for error AND exception is better.
Is it really usefull for end-user to know that an error occured, or and exception? Once again, i'm not really sure.